src/services/FeedProcessor.ts

```diff
@@ -123,9 +129,25 @@ class FeedProcessor {
       return;
     }
 
+    let feedBase = '';
```

Comment:
put logic for base/quote in separate method (or some class) and remembers about `nl`

Reply:
done